src/BenchmarksApps/HttpClientBenchmarks/Servers/Kestrel/Program.cs

```diff
@@ -0,0 +1,37 @@
+using System.Security.Cryptography;
+using System.Security.Cryptography.X509Certificates;
+using Microsoft.AspNetCore.Server.Kestrel.Core;
+
+var builder = WebApplication.CreateBuilder(args);
+builder.WebHost.ConfigureKestrel(serverOptions =>
+{
+    serverOptions.ConfigureEndpointDefaults(listenOptions => 
+    {
+        // Create self-signed cert for server.
+        using (RSA rsa = RSA.Create())
+        {
+            var certReq = new CertificateRequest("CN=contoso.com", rsa, HashAlgorithmName.SHA256, RSASignaturePadding.Pkcs1);
+            certReq.CertificateExtensions.Add(new X509BasicConstraintsExtension(false, false, 0, false));
+            certReq.CertificateExtensions.Add(new X509EnhancedKeyUsageExtension(new OidCollection { new Oid("1.3.6.1.5.5.7.3.1") }, false));
+            certReq.CertificateExtensions.Add(new X509KeyUsageExtension(X509KeyUsageFlags.DigitalSignature, false));
+            X509Certificate2 cert = certReq.CreateSelfSigned(DateTimeOffset.UtcNow.AddMonths(-1), DateTimeOffset.UtcNow.AddMonths(1));
+            if (OperatingSystem.IsWindows())
+            {
+                cert = new X509Certificate2(cert.Export(X509ContentType.Pfx));
+            }
+            listenOptions.UseHttps(cert);
+        }
+        listenOptions.Protocols = HttpProtocols.Http1AndHttp2AndHttp3;
+    });
+});
+var app = builder.Build();
+
+app.MapGet("/", () => Results.Ok());
+
+app.MapGet("/get", () => "Hello World!");
+
+await app.StartAsync();
+
+Console.WriteLine("Application started"); // readyStateText
```

Comment:
The default logging should already output that. But doesn't hurt.

Reply:
Ah right, there's `Application started. Press Ctrl+C to shut down.`. It was in the middle of other default logs so I missed that. And my `readyStateText` said something else at first, so it wasn't working, and only after I got it working I changed it back to "Application started" 😄